README.md

```diff
@@ -85,6 +85,7 @@ Currently supported data connectors for upstream datasets. More coming soon.
 | `flightsql`   | Apache Arrow Flight SQL                                                                        | Alpha  | Arrow Flight SQL                                                                                   |
 | `snowflake`   | Snowflake                                                                                      | Alpha  | Arrow                                                                                              |
 | `ftp`, `sftp` | FTP/SFTP                                                                                       | Alpha  | Parquet, CSV                                                                                       |
+| `graphql`     | GraphQL                                                                                        | Alpha  | GraphQL                                                                                            |
```

Comment:
Lets not include this in the README.md until we are ready for it to be released to users.